Helm/opensearch/values.yaml

```diff
@@ -0,0 +1,295 @@
+---
+clusterName: "opensearch-cluster"
+nodeGroup: "master"
+
+# The service that non master groups will try to connect to when joining the cluster
+# This should be set to clusterName + "-" + nodeGroup for your master group
+masterService: "opensearch-cluster-master"
+
+# OpenSearch roles that will be applied to this nodeGroup
+# These will be set as environment variables. E.g. node.master=true
+roles:
+  master: "true"
+  ingest: "true"
+  data: "true"
+  remote_cluster_client: "true"
+
+replicas: 2
+minimumMasterNodes: 1
+
+majorVersion: ""
+
+
+# Allows you to add any config files in /usr/share/opensearch/config/
+# such as opensearch.yml and log4j2.properties
+config:
+#  opensearch.yml: |
+#    xpack.security.enabled: false
+#    xpack.security.transport.ssl.enabled: false
+#    xpack.security.transport.ssl.verification_mode: certificate
+#    xpack.security.transport.ssl.keystore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.transport.ssl.truststore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.http.ssl.enabled: false
+#    xpack.security.http.ssl.truststore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+#    xpack.security.http.ssl.keystore.path: /usr/share/opensearch/config/certs/opensearch-certificates.p12
+
+
+# Extra environment variables to append to this nodeGroup
+# This will be appended to the current 'env:' key. You can use any of the kubernetes env
+# syntax here
+extraEnvs: []
+#  - name: MY_ENVIRONMENT_VAR
+#    value: the_value_goes_here
+
+# Allows you to load environment variables from kubernextes secret or config map
+envFrom: []
+# - secretRef:
+#     name: env-secret
+# - configMapRef:
+#     name: config-map
+
+# A list of secrets and their paths to mount inside the pod
+# This is useful for mounting certificates for security and for mounting
+# the X-Pack license
+secretMounts: []
+
+
+hostAliases: []
+#- ip: "127.0.0.1"
+#  hostnames:
+#  - "foo.local"
+#  - "bar.local"
+
+image: "opensearchproject/opensearch"
+imageTag: "latest"
+imagePullPolicy: "IfNotPresent"
+
+podAnnotations: {}
+  # iam.amazonaws.com/role: es-cluster
+
+# additionals labels
+labels: {}
+
+esJavaOpts: "-Xmx512M -Xms512M"
+
+resources:
+  requests:
+    cpu: "1000m"
+    memory: "100Mi"
+
+initResources: {}
+  # limits:
+  #   cpu: "25m"
+  #   # memory: "128Mi"
+  # requests:
+  #   cpu: "25m"
+  #   memory: "128Mi"
+
+sidecarResources: {}
+  # limits:
+  #   cpu: "25m"
+  #   # memory: "128Mi"
+  # requests:
+  #   cpu: "25m"
+  #   memory: "128Mi"
+
+networkHost: "0.0.0.0"
+
+volumeClaimTemplate:
+  accessModes: [ "ReadWriteOnce" ]
+  resources:
+    requests:
+      storage: 500M
```

Comment:
Why is `volumeClaimTemplate` a dedicated section and `persistence` another section? 

Should the content of `persistence` populate the template?

Reply:
`volumeClaimTemplate` will help adjust the storage and storageClass and `persistence` is just for enabling the persistent volume.

Reply:
Look at other chart examples. The "traditional" approach to this I'd having persistence manage size, storageclass and other bits. Seeing as there really isn't much to a claim template